### PR TITLE
Add ACME client counts to vault operator usage

### PR DESCRIPTION
**Pull request by raskchanky**
https://hashicorp.atlassian.net/browse/VAULT-24578

### CODE REVIEW

File: command/operator_usage.go [hunk 1]
```diff
@@ -245,6 +246,11 @@ func (c *OperatorUsageCommand) parseNamespaceCount(rawVal interface{}) (UsageRes
 	// don't error if the secret syncs key is missing
 	ret.secretSyncs, _ = jsonNumberOK(counts, "secret_syncs")
 
+	ret.acmeCount, ok = jsonNumberOK(counts, "acme_clients")
+	if !ok {
+		return ret, errors.New("missing acme_clients")
```
Comment (miagilepner) [inline command/operator_usage.go:251]: I don't think we should error here - if the vault client version is greater than the vault server version, then `acme_clients` might be missing from the response

File: command/operator_usage.go [hunk 2]
```diff
@@ -321,13 +327,19 @@ func (c *OperatorUsageCommand) totalOutput(data map[string]interface{}) []string
 	// don't error if secret syncs key is missing
 	secretSyncs, _ := jsonNumberOK(total, "secret_syncs")
 
+	acmeCount, ok := jsonNumberOK(total, "acme_clients")
+	if !ok {
+		c.UI.Error("missing acme_clients in total")
```
Comment (miagilepner) [inline command/operator_usage.go:332]: same here, this probably doesn't require an error

File: command/operator_usage.go (hunk 1 above)
Comment (raskchanky) [reply command/operator_usage.go:251]: Ah good point

**Review by miagilepner — APPROVED** [review]
LGTM!

**Pull request merged**